index.html

```diff
@@ -301,6 +451,6 @@ <h6 class="first-par">I'm always interested in hearing about new projects, so if
            </div>
        </footer>
```

Comment:
- [ ] In the last part there is two contact form so kindly remove the last one.
### your contact form
![hiiiiss](https://user-images.githubusercontent.com/81727988/173113145-9f94b60b-16c5-46cc-b9e4-4005145b699d.png)